keras_nlp/models/opt/opt_backbone_test.py

```diff
@@ -91,6 +95,22 @@ def test_saved_model(self, save_format, filename):
         restored_output = restored_model(self.input_batch)
         self.assertAllClose(model_output, restored_output)
 
+    def test_create_layout_map(self):
+        mesh = tf.experimental.dtensor.create_mesh([("batch", 1), ("model", 1)])
+        with OPTBackbone.create_layout_map(mesh).scope():
+            OPTBackbone(
+                vocabulary_size=10,
+                num_layers=2,
+                num_heads=2,
+                hidden_dim=2,
+                intermediate_dim=4,
+                max_sequence_length=5,
+            )
+        # Using DTensor enables the mlir bridge as a side effect. Eventually
+        # this will be default, but for now we face with the bridge and must
```

Comment:
we face with the bridge?

Reply:
oops, not sure where my mind was at. updated.